modules/websocket/wsl_server.cpp

```diff
@@ -43,7 +43,7 @@ WSLServer::PendingPeer::PendingPeer() {
 	memset(req_buf, 0, sizeof(req_buf));
 }
 
-bool WSLServer::PendingPeer::_parse_request(const Vector<String> p_protocols) {
+bool WSLServer::PendingPeer::_parse_request(const Vector<String> p_protocols, String &resource_name) {
```

Comment:
As it's passed as a reference, the parameter should be named `r_resource_name` (same in `do_handshake`).

Reply:
I take it the original variable `resource_name` should **not** get the `r_` up until and including the point where it appears as an argument in the first call?

Reply:
Yeah it's only in the function prototype, and thus in the function itself. But in the calling function where it's a local variable, it takes no prefix (same thing for `p_` read-only parameters).